test/tests/AverageExtensionRatio/tests

```diff
@@ -0,0 +1,24 @@
+[Tests]
+  issues = '#251'
+  design = 'AverageExtensionRatio.md'
+
+  [AverageExtensionRatio]
+    requirement = "AverageExtensionRatio computes the average extension ratio between two sets of points specified in two arrays: `first_point` and `last_point`. It should result in error when sizes of `first_point` and `last_point` do not match and size of `first_point` is zero "
```

Comment:
A few minor issues here:
1) There doesn't appear to be a test of the postprocessor under standard conditions.
2) If you're going to re-use the same -i file like you're doing for the error tests (which is a good idea), you should add `prereq` so they don't all run at the same time.
3) The way you have the requirements specified isn't really the best.  I think I wouldn't use `detail` in this case. I'd just have three tests with three requirements:
- The system shall provide a postprocessor that computes the average extension ratio between two sets of points.
- AverageExtensionRatio shall generate an error if the sizes of the specified sets of points do not match.
- AverageExtensionRatio shall generate an error if the size of either of the sets of points is zero.